README.md

```diff
@@ -82,6 +82,38 @@ const client = await search()
 
 Now `client` is an instance of the [OpenSearch JavaScript client](https://opensearch.org/docs/latest/clients/javascript/index/), and you can call any client method on it — for example, [`client.index.create()`](https://opensearch.org/docs/latest/clients/javascript/index/#creating-an-index), [`client.index()`](https://opensearch.org/docs/latest/clients/javascript/index/#indexing-a-document), or [`client.search()`](https://opensearch.org/docs/latest/clients/javascript/index/#searching-for-documents).
 
+## Making post-deployment requests to OpenSearch or ElasticSearch from your application
+
+If you would like to make requests to automatically configure your OpenSearch or ElasticSearch instance after deployment, you may optionally add a postdeploy-search.js file in the root directory of your Architect project. This file should export a function that takes no arguments as its default export. Be sure to connect with your OpenSearch or ElasticSearch instance before making your requests.
```

Comment:
Rather than "automatically configure" let's explain this as post-deployment REST API calls - which can include both cluster configuration and adjusting indices (the latter being your intended application).